scripts/install-mender.sh

```diff
@@ -202,7 +202,7 @@ get_deps() {
 add_repo() {
     curl -fsSL $REPO_URL/gpg | apt-key add -
 
-    repo="deb [arch=$ARCH] $REPO_URL $CHANNEL main"
+    repo="deb [arch=$ARCH] $REPO_URL $DIST_VERSION/$CHANNEL main"
```

Comment:
As we informally agreed on Slack, we need here first to check if the old repo exists in `/etc/apt/sources.list.d/mender.list`. If so, we print an error message and give instructions to how to manually remove it (`sudo rm /etc/apt/sources.list.d/mender.list`) and ask to run again the script.